pom.xml

```diff
@@ -481,6 +481,17 @@
       <artifactId>zookeeper</artifactId>
       <version>${zookeeper.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>${kafka.version}</version>
```

Comment:
shouldn't this be only for test?

Reply:
Yes, it can be. Not having the dep at all fails the build. I thought it should maybe a first level dependency since KTB is using the api directly.

As it is now KTB is getting this dep via:
<dependency>
      <groupId>io.confluent</groupId>
      <artifactId>kafka-schema-registry-client</artifactId>
    </dependency>

Not sure. What do you think?

Reply:
fair point, and good finding here. I did a quick look and it looks it getting the necessary deps from two possible places:

```bash
[INFO] +- io.confluent:kafka-client-plugins:jar:5.5.1-ce:compile
[INFO] |  +- (org.apache.kafka:kafka-clients:jar:5.5.1-ce:compile - omitted for conflict with 5.5.0-ccs)
[INFO] |  +- io.confluent:rest-authorizer:jar:5.5.1-ce:compile
[INFO] |  |  +- (org.apache.kafka:kafka-clients:jar:5.5.1-ce:compile - omitted for conflict with 5.5.0-ccs)
[INFO] |  |  +- io.confluent:authorizer:jar:5.5.1-ce:compile
[INFO] |  |  |  +- (org.apache.kafka:kafka-clients:jar:5.5.1-ce:compile - omitted for conflict with 5.5.0-ccs)
[INFO] |  |  |  +- (com.fasterxml.jackson.core:jackson-databind:jar:2.10.2:compile - omitted for conflict with 2.7.9.5)
[INFO] |  |  |  +- (com.fasterxml.jackson.datatype:jackson-datatype-jdk8:jar:2.10.2:compile - omitted for duplicate)
[INFO] |  |  |  \- (javax.ws.rs:javax.ws.rs-api:jar:2.1.1:compile - omitted for duplicate)
```
^^ skipped in favor of 

```bash
[INFO] +- io.confluent:kafka-schema-registry-client:jar:5.5.0:compile
[INFO] |  +- org.apache.kafka:kafka-clients:jar:5.5.0-ccs:compile
[INFO] |  |  +- com.github.luben:zstd-jni:jar:1.4.4-7:compile
[INFO] |  |  +- org.lz4:lz4-java:jar:1.7.1:compile
[INFO] |  |  +- org.xerial.snappy:snappy-java:jar:1.1.7.3:compile
```
when it got introduced.

so +1 on keeping it. However please update the _kafka.version_ property (in the pom file) to be _2.5.0_ please, like this the admin client is at least equal the one used earlier.

Reply:
Yes, will do. So version 5.5.0-ccs = 2.5.0 ?